src/platform/packages/shared/kbn-unified-data-table/src/components/data_table.tsx

```diff
@@ -1058,11 +1057,10 @@ export const UnifiedDataTable = ({
 
     return {
       allowDensity: Boolean(onUpdateDataGridDensity),
-      allowRowHeight: false,
+      allowRowHeight: Boolean(onChangeRowHeight) || Boolean(onChangeHeaderRowHeight),
```

Comment:
When we render all those controls from `customRender` we need to allow and `rowHeight` controls conditionally (before `rowHeight` was set to false and rendered via `additionalDisplaySettings`), hence this line